javagen/src/main/java/com/azure/autorest/mapper/ProxyMethodMapper.java

```diff
@@ -538,7 +539,10 @@ private static SwaggerExceptionDefinitions getSwaggerExceptionDefinitions(Operat
                     }
                 }
 
-                if (swaggerDefaultExceptionType == null) {
+                if (swaggerDefaultExceptionType == null
+                        && !CoreUtils.isNullOrEmpty(operation.getExceptions())
+                        // m4 could return Response without schema, when the Swagger uses e.g. "produces: [ application/x-rdp ]"
+                        && operation.getExceptions().get(0).getSchema() != null) {
                     // no default error, use the 1st to keep backward compatibility
                     swaggerDefaultExceptionType = processExceptionClassType(
                             (ClassType) Mappers.getSchemaMapper().map(operation.getExceptions().get(0).getSchema()), settings);
```

Comment:
Add a check on whether the `exceptions.[0].schema` is `null`. Skip the SchemaMapper if schema is null.
(if we do the map, `swaggerDefaultExceptionType` will still be `null`)

Existing code will fill the blank
https://github.com/Azure/autorest.java/blob/68190ca3926755c36e10a0b9165f7584b7bc31ad/javagen/src/main/java/com/azure/autorest/mapper/ProxyMethodMapper.java#L458-L462